src/DDTrace/Integrations/IntegrationsLoader.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace DDTrace\Integrations;
+
+use DDTrace\Configuration;
+use DDTrace\Integrations\Curl\CurlIntegration;
+use DDTrace\Integrations\ElasticSearch\V1\ElasticSearchIntegration;
+use DDTrace\Integrations\Eloquent\EloquentIntegration;
+use DDTrace\Integrations\Guzzle\V5\GuzzleIntegration;
+use DDTrace\Integrations\Memcached\MemcachedIntegration;
+use DDTrace\Integrations\Mysqli\MysqliIntegration;
+use DDTrace\Integrations\PDO\PDOIntegration;
+use DDTrace\Integrations\Predis\PredisIntegration;
+
+/**
+ * Loader for all integrations currently enabled.
+ */
+class IntegrationsLoader
+{
+    const LIBRARIES = [
+        CurlIntegration::NAME => '\DDTrace\Integrations\Curl\CurlIntegration',
+        ElasticSearchIntegration::NAME => '\DDTrace\Integrations\ElasticSearch\V1\ElasticSearchIntegration',
+        EloquentIntegration::NAME => '\DDTrace\Integrations\Eloquent\EloquentIntegration',
+        GuzzleIntegration::NAME => '\DDTrace\Integrations\Guzzle\V5\GuzzleIntegration',
+        MemcachedIntegration::NAME => '\DDTrace\Integrations\Memcached\MemcachedIntegration',
+        MysqliIntegration::NAME => '\DDTrace\Integrations\Mysqli\MysqliIntegration',
+        PDOIntegration::NAME => '\DDTrace\Integrations\PDO\PDOIntegration',
+        PredisIntegration::NAME => '\DDTrace\Integrations\Predis\PredisIntegration',
+    ];
+
+    /**
+     * Loads all the enabled library integrations.
+     */
+    public static function load()
+    {
+        $globalConfig = Configuration::get();
+
+        if (!$globalConfig->isEnabled()) {
+            return;
+        }
+
+        if (!extension_loaded('ddtrace')) {
+            error_log('Missing ddtrace extension. To disable tracing set env variable DD_TRACE_ENABLED=false');
```

Comment:
would - since we allow now for things to be disabled `$globalConfig->isEnabled()` - would it make sense to bring `E_USER_WARNING` back instead of `error_log?

Sort of makes sense i think, especially since we're adding instructions how to disable this via DD_TRACE_ENABLED=false
WDYT ?

Reply:
This is a good idea actually, let me do it very quick!